tests/alpha.model.tests/src/alpha/model/tests/util/FaceLatticeTest.xtend

```diff
@@ -98,6 +98,11 @@ class FaceLatticeTest {
 	 * exactly match the vertices that ISL calculates for the same set.
 	 */
 	def private static assertVerticesMatchISL(FaceLattice lattice) {
```

Comment:
This solution looks good to me. For future reference (in case anyone reads this in the future): when isl computes the vertices of the basic sets that make up the 0-dimensional faces of the lattice, it returns multiple chambers, which the remainder of the test doesn't really handle. A good enough workaround is to check that the lattice has the correct number of nodes at each layer.